### PR TITLE
ci: run interop tests in CI

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

File: .github/workflows/go-interop.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+on: pull_request
+name: Interop
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    name: GO/JS
```
Comment (marten-seemann) [inline .github/workflows/go-interop.yml:7]:
```suggestion
    name: Go/JS
```

File: .github/workflows/go-interop.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        working-directory: interop/go
+        run: go build -o test-go .
+      - name: Test
+        working-directory: interop
+        run: |
+          npm --prefix ./js/ start &
+          sleep 1
+          ./go/test-go &
+          wait -n
+          wait -n
```
Comment (marten-seemann) [inline .github/workflows/go-interop.yml:30]: Why do we need this twice? iiuc `wait -n` waits for all background jobs to complete?

Comment (Stebalien) [reply .github/workflows/go-interop.yml:30]: `wait -n` waits for the next background job to complete and returns its exit code. Unfortunately, a simple `wait` won't catch the exit code.

Comment (Stebalien) [reply .github/workflows/go-interop.yml:30]: I'll add a comment.

**Review by marten-seemann — APPROVED** [review]

**Pull request merged**